notion-distcopy/src/test/scala/com/ambiata/notion/distcopy/DistCopySpec.scala

```diff
@@ -28,7 +28,7 @@ class DistCopySpec extends AwsScalaCheckSpec(tests = 5) { def is = s2"""
    the full path of the file to copy $createDownloadMappingForADirectory
 
   An upload mapping must contain
-   the full path of the file to copy $createUploadMappingForADirectory
+    the path of the file to copy relative to the from dir $createUploadMappingForADirectory
```

Comment:
I added an example in the `DistCopy` class to explain and motivate the current behaviour.